{
	"id": "ALSA-2024:0114",
	"advisories": [
		{
			"id": "ALSA-2024:0114",
			"title": "Moderate: python3 security update",
			"description": "Python is an interpreted, interactive, object-oriented programming language, which includes modules, classes, exceptions, very high level dynamic data types and dynamic typing. Python supports interfaces to many system calls and libraries, as well as to various windowing systems.\n\nSecurity Fix(es):\n\n* python: use after free in heappushpop() of heapq module (CVE-2022-48560)\n* python: DoS when processing malformed Apple Property List files in binary format (CVE-2022-48564)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
			"severity": [
				{
					"type": "vendor",
					"source": "errata.almalinux.org",
					"vendor": "Moderate"
				}
			],
			"references": [
				{
					"source": "errata.almalinux.org",
					"url": "https://bugzilla.redhat.com/2249755"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://errata.almalinux.org/8/ALSA-2024-0114.html"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/errata/RHSA-2024:0114"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2022-48560"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2022-48564"
				},
				{
					"source": "errata.almalinux.org",
					"url": "https://bugzilla.redhat.com/2249750"
				}
			],
			"published": "2024-01-10T00:00:00Z",
			"modified": "2024-01-16T20:00:16Z"
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2022-48560",
			"references": [
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2022-48560"
				}
			]
		},
		{
			"id": "CVE-2022-48564",
			"references": [
				{
					"source": "errata.almalinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2022-48564"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python3-idle",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python3-libs",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python3-test",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "python3-tkinter",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "platform-python-devel",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "platform-python-debug",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		},
		{
			"ecosystem": "alma:8",
			"vulnerable": true,
			"package": {
				"name": "platform-python",
				"architectures": [
					"i686",
					"x86_64",
					"aarch64",
					"ppc64le",
					"s390x"
				]
			},
			"affected": {
				"type": "rpm",
				"range": [
					{
						"lt": "0:3.6.8-56.el8_9.2.alma.1"
					}
				],
				"fixed": [
					"0:3.6.8-56.el8_9.2.alma.1"
				]
			}
		}
	],
	"data_source": "alma-errata"
}
